From 9b2c0540028de7f991b134bffb1131410fba0b38 Mon Sep 17 00:00:00 2001
From: Jee-Yong Um <jc9.um@samsung.com>
Date: Mon, 16 Nov 2015 13:01:42 +0900
Subject: enventor: remove --eo option from eolian_helper

Summary: remove --eo option from Makefile_Eolian_Helper.am to fix build fail

Reviewers: Hermet, Jaehyun_Cho

Reviewed By: Jaehyun_Cho

Differential Revision: https://phab.enlightenment.org/D3331
---
 Makefile_Eolian_Helper.am | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile_Eolian_Helper.am b/Makefile_Eolian_Helper.am
index 168f659..377f173 100644
--- a/Makefile_Eolian_Helper.am
+++ b/Makefile_Eolian_Helper.am
@@ -13,10 +13,10 @@ am__v_EOL_0 = @echo "  EOLIAN  " $@;
 SUFFIXES = .eo .eo.c .eo.h .eo.legacy.h
 
 %.eo.c: %.eo ${_EOLIAN_GEN_DEP}
-	$(AM_V_EOL)$(EOLIAN_GEN) --eo --legacy $(EOLIAN_FLAGS) --gc -o $@ $<
+	$(AM_V_EOL)$(EOLIAN_GEN) --legacy $(EOLIAN_FLAGS) --gc -o $@ $<
 
 %.eo.h: %.eo ${_EOLIAN_GEN_DEP}
-	$(AM_V_EOL)$(EOLIAN_GEN) --eo $(EOLIAN_FLAGS) --gh -o $@ $<
+	$(AM_V_EOL)$(EOLIAN_GEN) $(EOLIAN_FLAGS) --gh -o $@ $<
 
 %.eo.legacy.h: %.eo ${_EOLIAN_GEN_DEP}
 	$(AM_V_EOL)$(EOLIAN_GEN) --legacy $(EOLIAN_FLAGS) --gh -o $@ $<
-- 
cgit v0.12


